atala-prism-sdk/src/commonMain/kotlin/io/iohk/atala/prism/walletsdk/prismagent/ConnectionManager.kt

```diff
@@ -13,6 +13,24 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.first
 import kotlin.jvm.Throws
 
+interface ConnectionManager : ConnectionsManager, DIDCommConnection {
+
+    val mediationHandler: MediationHandler
+
+    suspend fun startMediator()
+    suspend fun registerMediator(host: DID)
+
+    override suspend fun addConnection(paired: DIDPair)
+
+    override suspend fun removeConnection(pair: DIDPair): DIDPair?
+
+    override suspend fun awaitMessages(): Flow<Array<Pair<String, Message>>>
+
+    override suspend fun awaitMessageResponse(id: String): Message?
+
+    override suspend fun sendMessage(message: Message): Message?
+}
+
```

Comment:
Did you need to add this now? :D I know it's in Swift but it's a component I want to delete.

Reply:
I created this interface to facilitate unit tests and mocking of this component.